src/styles/_catalog.scss

```
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+  
+    &:hover {
+      color: #34568b;
+      border-radius: 20%;
+      transition: 0.3s;
```

```suggestion
      transition: 0.3s;
```
to work both ways, transition property should be applied not on :hover but on element itself.
Also, tou should always specify the `transition-property`
https://developer.mozilla.org/en-US/docs/Web/CSS/transition-property